### ISSUE TITLE
matlab wrapper broken

### ISSUE THREAD

**Issue by elhuhdron** [scalableminds/webknossos-wrap#44]
It seems some change made to the rust interface was not added to the matlab wrapper, as wkwBuild() is failing for me in the current HEAD. I fixed this by adding false for the C-order flag to the Mat constructor call:

```
diff --git a/matlab/rust/wkw_mex/src/wkw.rs b/matlab/rust/wkw_mex/src/wkw.rs
index 8e0f71e..060f95c 100644
--- a/matlab/rust/wkw_mex/src/wkw.rs
+++ b/matlab/rust/wkw_mex/src/wkw.rs
@@ -98,5 +98,5 @@ pub fn mx_array_mut_to_wkwrap_mat<'a>(
     // voxel type
     let voxel_type = mx_class_id_to_voxel_type(unsafe { mxGetClassID(pm) })?;
 
-    wkwrap::Mat::new(buf, shape, voxel_size, voxel_type)
+    wkwrap::Mat::new(buf, shape, voxel_size, voxel_type, false)
 }
```

Then it builds, and it worked at least for one test case that I have for reading a wkw roi. Can you please incorporate the change, or fix appropriately if my fix is not correct? I can submit a pull request if desired.
Thank You!

**Comment by amotta** [comment]
Thanks you very much for the bug report and the patch. I've just created a pull request with you as co-author (after a few failed attempts).

Builds with `wkwBuild` now work and tests (`wkwTest` and `wkwCompression`) all pass.

**Comment by amotta** [comment]
Fixed by commit 938f9f0ee706d65e92e4f5a1e23c7356c97d7ff0
